### ISSUE TITLE
Is gonb support table?

### ISSUE THREAD

**Issue by ChengYen-Tang** [janpfeifer/gonb#137]
Just like python pandas, when I print the dataframe, Jupyter will display a table
https://github.com/pandas-dev/pandas/issues/16792

**Comment by janpfeifer** [comment]
Yes, this is high on my wish-list for gonb. It shouldn't be that hard, except, which Go library to support data views should one use ? Any suggestions ?

**Comment by ChengYen-Tang** [comment]
I don't have any idea either😅

**Comment by janpfeifer** [comment]
Let's keep this open. I'm sure something is going to come up, and then we add it. But definitely this is something that would be very helpful to have.

**Comment by ChengYen-Tang** [comment]
Other questions,
```
!*go mod edit -replace "test=/mnt/f/test"
!*cat go.mod
```
I expect to be able to use pwd, similar to the following
```
!*go mod edit -replace "test=$(pwd)"
!*cat go.mod
```
But, $(pwd) = tmp/xxx
I konw, because I uesd '!*'

**Comment by janpfeifer** [comment]
Ha, so it's good you already understood that when `!*` is used, it changes the directory to the temporary cell code directory.

Now the directory with the cell code is also available in the `${GONB_TMP_DIR}` variable. So what you could do is:

```
!dd=$(pwd); cd ${GONB_TMP_DIR} ; go mod edit -replace "test=${dd}"; cat go.mod
```

**Comment by janpfeifer** [comment]
Oh, btw, GoNB also sets `$GONB_DIR` with the directory where the cells are being run from. So a simpler alternative would be:

```
!*go mod edit -replace "test=${GONB+DIR}"
!*cat go.mod
```
